routes/users.js

```diff
@@ -1,9 +1,15 @@
-const express = require('express');
-const router = express.Router();
+const express = require('express')
+const User = require('../models/user.js')
```

Comment:
Let's not change the order of existing imports - it introduces changes in the diff that aren't really changes.  :)  Just shift the `User` import down below the instantiation of `router` and it should be good